libioc/Jail.py

```diff
@@ -630,27 +633,40 @@ def __mount_devfs(
 
         yield event.end()
 
-    def __unmount_devfs(
+    def __mount_fdescfs(
         self,
```

Comment:
is this just git diff weirdness, or why did `__umount_devfs` go?

Reply:
__umount_devfs was unused because Storage.teardown already unmounts `/dev/fd` and `/dev`.